DXRModules/DeusEx/Classes/DXREventsBase.uc

```diff
@@ -99,8 +101,9 @@ function PostFirstEntry()
 {
     Super.PostFirstEntry();
 
-    MarkBingoFailedSpecial();
     MarkBingoFailedGeneric();
```

Comment:
Couldn't all of this logic be implemented in the generic check by simply masking out missions beyond the end mission?